ext/spl/tests/bug51068.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+SPL: glob wrapper interactions with DirectoryIterator
+--SKIPIF--
+<?php if (!extension_loaded("spl")) print "skip"; ?>
```

Comment:
This SKIPIF section is useless as the SPL extension is always loaded and can’t be disabled.

Reply:
Fixed but "pull" bot closed the MR not sure why 

anyway I reopened it here with the changes
https://github.com/php/php-src/pull/3771